dcm2bids/structure.py

```diff
@@ -217,27 +217,47 @@ def intendedFor(self, value):
         else:
             self._intendedFor = [value]
 
-    def dstSidecarData(self, descriptions):
+    def dstSidecarData(self, descriptions, descriptionMap):
         """
         """
         data = self.srcSidecar.origData
         data["Dcm2bidsVersion"] = __version__
 
         # intendedFor key
+        # Check if intendedFor is None
         if self.intendedFor != [None]:
             intendedValue = []
             for index in self.intendedFor:
```

Comment:
Maybe use another variable name here since it can be either an index or string

Reply:
Addressed in 3f0793d45358bfb6aa311f721159a923c2af6674